### ISSUE TITLE
Fill out services documentation for FE FHIR approach

### ISSUE THREAD

**Issue by jbalboni** [department-of-veterans-affairs/va.gov-team#7717]
We need to document the module and function organization of the "services" layer that we'll use to switch between the FHIR apis and the old VAR apis.

Started at https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/engineering/discovery/FHIR/fe_service_design.md

This should cover:

- Endpoints called in each function (FHIR and VAR)
- Function parameters and output data (at a high level, schemas will be defined in a separate ticket)

**Comment by lenaecb** [comment]
Blocked by all the things

**Comment by narin** [comment]
Updated the service doc and created new pages for the services we have worked on.

Created a [separate ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9730) to document HealthcareService and Schedule in the future.
